Quadriga/src/main/java/edu/asu/spring/quadriga/aspects/annotations/GetProject.java

```diff
@@ -0,0 +1,12 @@
+package edu.asu.spring.quadriga.aspects.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.PARAMETER)
+public @interface GetProject {
```

Comment:
I would call this annotation ```ProjectIdentifier``` as the parameter annotated with this identifies the project that needs to be injected.